VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/copy-quick-link/copy-quick-link.component.ts

```diff
@@ -21,7 +21,12 @@ export class CopyQuickLinkComponent implements OnInit {
     async ngOnInit() {
         this.tooltip = this.translateService.instant('copy-quick-link.display-text');
         const response = await this.vhoQueryService.getConferenceByIdVHO(this.conferenceId);
-        this.hearingId = response && response.hearing_id;
+        this.hearingId = response.hearing_id;
+    }
+
+    async ngOnChanges(): Promise<void> {
+        const response = await this.vhoQueryService.getConferenceByIdVHO(this.conferenceId);
```

Comment:
Why do we need to call the API this in this component?

Reply:
The HearingId is not present in the HearingSummary object we are passing from its parent component. We need to retrieve it.

Reply:
Ahhh. I see. I didn't notice that this was in the ngOnInit as well. Should this ever change though?

Also is there any chance we could add the hearing id to that model? It seems like it would simplify this a lot.